_config.yml

```diff
@@ -1,8 +1,16 @@
-# jekyll settings
+# Local Jekyll settings
```

Comment:
Is this shared config for prod, dev, and staging? or just dev (default)?

If the latter, is there a way to combine/merge configs so we don't have to duplicate settings (e.g. asset paths) across multiple files?

Reply:
Quick googling shows we might be able to do something like this: https://stackoverflow.com/questions/30897871/jekyll-config-multiple-files#30899655

-> maybe have a `common_config.yml` file with base settings, then use in combination with `staging...`, `prod...` etc?